src/ensembl/stories/shared-components/badged-button/BadgedButton.stories.scss

```diff
@@ -0,0 +1,21 @@
+@import 'src/styles/common';
+
+.wrapper {
+  padding: 40px;
+}
+
+.active {
+  border: none;
+  background-color: $ens-dark-grey;
+  color: $ens-white;
+}
+
+.imageButtonWrapper {
```

Comment:
why **image**Button?

Reply:
I'm trying to display a story with all the available buttons to make use of BadgedButton.

`imageButtonWrapper` styling is just used in the story for imageButton.

Reply:
Ah, I see.